src/app/shared/components/List/index.tsx

```diff
@@ -210,48 +258,23 @@ ListProps<T, F>): JSX.Element {
       id="generic-list-component"
       className={`relative flex h-full flex-col overflow-x-hidden overflow-y-hidden ${className}`}
     >
-      {/* HEAD */}
-      <div className="flex h-12 shrink-0 flex-row px-5">
-        {/* COLUMN */}
-        <div className="flex h-full min-w-full flex-row items-center border-b border-gray-10">
-          {/* SELECTION CHECKBOX */}
-          <div className="flex h-full flex-row items-center justify-between pr-4">
-            <BaseCheckbox
-              checked={selectedItems.length > 0}
-              indeterminate={items.length > selectedItems.length && selectedItems.length > 0}
-              onClick={onTopSelectionCheckboxClick}
-            />
-          </div>
-
-          {header.map((column) => (
-            <div
-              onClick={column.orderable ? () => onOrderableColumnClicked(column) : undefined}
-              key={column.name.toString()}
-              className={`flex h-full shrink-0  flex-row items-center space-x-1.5 text-base font-medium text-gray-60  ${
-                column.width
-              } ${column.orderable ? 'cursor-pointer hover:text-gray-80' : ''}`}
-            >
-              <span>{column.label}</span>
-              {column.name === orderBy?.field &&
-                column.orderable &&
-                (orderBy?.direction === 'ASC' ? (
-                  <ArrowUp size={14} weight="bold" />
-                ) : (
-                  <ArrowDown size={14} weight="bold" />
-                ))}
-            </div>
-          ))}
-          {isVerticalScrollbarVisible && <div className="mr-15px" />}
-          {menu && <div className="flex h-full w-12 shrink-0" />}
-        </div>
-      </div>
-
-      {/* BODY */}
       <div id="scrollableList" className="flex h-full flex-col overflow-x-auto overflow-y-auto" ref={ref}>
         {(!hasMoreItems ?? false) && items.length === 0 && !isLoading ? (
           emptyState
         ) : items.length > 0 ? (
           <>
+            <Header
+              selectedItems={selectedItems}
+              onTopSelectionCheckboxClick={onTopSelectionCheckboxClick}
+              items={items}
+              header={header}
+              orderBy={orderBy}
+              onOrderableColumnClicked={onOrderableColumnClicked}
+              menu={menu}
+              isVerticalScrollbarVisible={isVerticalScrollbarVisible}
+            />
+
```

Comment:
In this way the header it will be scrolled. It has to stay fixed.
I think that we have to add the conditional to the Header component outside the Body